src/fontra/views/fontinfo/panel-development-status-definitions.js

```diff
@@ -3,34 +3,35 @@ import * as html from "../core/html-utils.js";
 import { addStyleSheet } from "../core/html-utils.js";
 import { enumerate, hexToRgba, range, rgbaToHex } from "../core/utils.js";
 import { BaseInfoPanel } from "./panel-base.js";
+import { translate } from "/core/localization.js";
 import { message } from "/web-components/modal-dialog.js";
 
 const defaultStatusFieldDefinitions = {
   "fontra.sourceStatusFieldDefinitions": [
     {
       color: [1, 0, 0, 1],
       isDefault: true,
-      label: "In progress",
+      labelKey: "development-status-definitions.default.label.in-progress",
```

Comment:
I don't think these should be translated: the status labels are stored in the font's customData, and I wouldn't want to add "labelKey" to the data structure. Also, when adding new custom statuses, we don't have the option to provide translations, so it'll be a bit half-assed anyway.

Reply:
Thanks. I reverted the changes relate to that.